### PR TITLE
Class 1 Slides Updates

### PR DESCRIPTION
**Pull request by caleybrock**
# What's in this PR?
All slide updates required for class 1

## Specific changes
- New graphs for comparing time and space complexity
- Updates to main page based on schedule
- Added a vocab sheet
- Modified code samples to use a repl so we can run code in class
- Other small updates to the slides
- Added some homework problems

## What's left to do for class 1?
- Practice to make sure timing makes sense

### CODE REVIEW

File: algorithms/class1.html [hunk 1]
```diff
@@ -187,7 +188,7 @@ <h2>Goal of Algorithms</h2>
                 <h2>Exercise time!</h2>
                 <h3 class='highlight'>What is an algorithm you use?</h3>
                 <br>
-                <p>Write down steps for several versions of an algorithm to solve an everyday problem</p>
+                <p>Find a partner and pick a problem. Come up with multiple versions of an algorithm and write them down.</p>
```
Comment (radmint) [inline algorithms/class1.html:191]: Might want to expand on what 'pick a problem' is. Perhaps everyday situation or something a bit more specific

File: algorithms/vocab1.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  	<table>
+	  <tr>
+	    <th>Concept</th>
+	    <th>Definition</th>
+	    <th style="width: 50%">Example</th>
+	  </tr>
+	  <tr>
+	    <td>Algorithm</td>
+	    <td>A repeatable process for determining the solution to a problem</td>
+	    <td></td>
```
Comment (radmint) [inline algorithms/vocab1.html:41]: Will you be adding examples?

File: index.html [hunk 3]
```diff
@@ -29,24 +29,37 @@ <h1 class="page-header">
       </header>
       <section class="lists">
           <div class="column">
+              <h3>Pre-work</h3>
```
Comment (radmint) [inline index.html:32]: 👍

File: algorithms/vocab1.html (hunk 2 above)
Comment (caleybrock) [reply algorithms/vocab1.html:41]: The plan was people could use this to take notes if they wanted to, so I was going to leave it blank.

File: algorithms/class1.html (hunk 1 above)
Comment (caleybrock) [reply algorithms/class1.html:191]: There are examples just below: https://github.com/gdiseattle/computer-science-fundamentals/pull/2/files#diff-86d3c912d2d3b2efc183261800c99d00R196

File: algorithms/vocab1.html (hunk 2 above)
Comment (radmint) [reply algorithms/vocab1.html:41]: Oh, interesting. Sounds good!

**Review by radmint — APPROVED** [review]

**Pull request merged**